descriptions/0/api.intercom.io.yaml

```diff
@@ -1492,6 +1492,296 @@ paths:
                   $ref: "#/components/schemas/away_status_reason"
         '401':
           "$ref": "#/components/responses/Unauthorized"
+  "/export/reporting_data/enqueue":
```

Comment:
It looks like this is already in place on the Unstable API. Is there a reason we need the nested resource name and a verb for the end point, rather than POST to `/reporting_data_export/` or similar to be RESTful?

Reply:
I see from this [comment](https://github.com/intercom/intercom/pull/390113/files#r2017000183) that this is following the pattern in Messages Data Export controller.